apps/manager_app/onboarding/onboarding.c

```diff
@@ -153,3 +154,28 @@ void onboarding_set_static_screen(void) {
 const flow_step_t *onboarding_get_step(void) {
   return &onboarding_flow;
 }
+
+onboarding_steps_e onboarding_get_last_step(void) {
+  // TODO: Set value in flash
```

Comment:
Why and would would we set in flash in a getter, seems like a missed comment.

Reply:
updated comments, yes left my mistake